js/index.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+
+	var cityAustin = true if 
```

Comment:
this `if` is gonna throw an error and prevent the rest of your application from running. Just remove it.